e2e/specs/password.spec.js

```diff
@@ -0,0 +1,31 @@
+import PortfolioPage from "../models/portfolioPage";
+import SettingsPage from "../models/settings/settingsPage";
+import GeneralSettingsPage from "../models/settings/generalSettingsPage";
+import PasswordEntryPage from "../models/passwordEntryPage";
+import { delay } from "../helpers";
+import { loadConfig } from "../bridge/server";
+
+const CORRECT_PASSWORD = "passWORD$123!";
+
+describe("Password Lock Screen", () => {
+  beforeAll(async () => {
+    await device.launchApp();
+  });
+
+  it("should be able to enter the correct password", async () => {
+    await loadConfig("1AccountBTC1AccountETH", true);
+
+    await PortfolioPage.waitForPageToBeVisible();
+    await PortfolioPage.navigateToSettings();
+    await SettingsPage.navigateToGeneralSettings();
+    await GeneralSettingsPage.togglePassword();
+    await GeneralSettingsPage.enterNewPassword(CORRECT_PASSWORD);
+    await GeneralSettingsPage.enterNewPassword(CORRECT_PASSWORD); // confirm password step
+    await device.sendToHome(); // leave LLM app and go to phone's home screen
+    await delay(60001); // password takes 60 seconds of app inactivity to activate
```

Comment:
You could override https://github.com/LedgerHQ/ledger-live-mobile/blob/develop/src/constants.js#L21 if in a test run to not have to wait I guess.